google-cloud-storage/src/main/java/com/google/cloud/storage/GrpcStorageImpl.java

```diff
@@ -936,12 +937,59 @@ public boolean deleteDefaultAcl(String bucket, Entity entity) {
 
   @Override
   public Acl createDefaultAcl(String bucket, Acl acl) {
-    return throwNotYetImplemented(fmtMethodName("createDefaultAcl", String.class, Acl.class));
+    return updateDefaultAcl(bucket, acl);
```

Comment:
why do we need to call update here? do we not have a create?

Reply:
The machinery of updating an acl or creating one is the same: 
1. Read Bucket metadata
2. possibly remove existing acl for entity
3. create new list from 2 + `acl`
4. Call update on bucket
5. Find new ObjectAccessControl in response
6. convert from proto to our model
7. return

Give that, I'm simply calling update

Reply:
understood.